pom.xml

```diff
@@ -146,21 +144,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
why is it removed? it is obligatory to be able to publish to jcenter / maven central repo

Reply:
My mistake removed temporarily since javadoc was not working on my machine due to setup, you can ignore any changes beyond the compiler change. I started looking at jd-core might prove to be a better alternative and it seems to support line numbers. Will let you know. Fernflower seems the most up to date, unfortunately didn't find any repository entry for it.

Reply:
*   https://github.com/JetBrains/intellij-community/tree/master/plugins/java-decompiler/engine
*   [https://github.com/fesh0r/fernflower](https://github.com/fesh0r/fernflower)